app/scripts/index.js

```diff
@@ -18,6 +18,49 @@ var _ = require('lodash');
 
   require('os-bootstrap/dist/js/os-bootstrap');
 
+  // Make config global available to app, and load scripts that require
+  // globalConfig.
+  $.get('./config')
+  .then(function(res) {
```

Comment:
It might be easier to have the path `config.json`, and then you'd run:

```javascript
$.get('config.json')  // the `./` is unecessary
  .then(function(config) {
    // The config will already have the logic of extracting env variables to their specific config keys

    globals.globalConfig = Object.assign({}, globals.globalConfig, config);

    return globals.globalConfig;
  });
```

The logic of extracting the environment variables to config keys would be implemented in the `config.json` route, similar to how it was done with the `config.js`.